src/core/Akka/Event/StandardOutLogger.cs

```diff
@@ -13,6 +13,38 @@
 
 namespace Akka.Event
 {
+    public abstract class MinimalLogger : MinimalActorRef
```

Comment:
MinimalLogger is the abstract base class for implementing minimal loggers.

Reply:
In a separate PR, we should add some documentation for this